src/main.cpp

```diff
@@ -2571,19 +2570,11 @@ bool ContextualCheckBlockHeader(const CBlockHeader& block, CValidationState& sta
     if (pcheckpoint && nHeight < pcheckpoint->nHeight)
         return state.DoS(100, error("%s: forked chain older than last checkpoint (height %d)", __func__, nHeight));
 
-    // Reject block.nVersion=1 blocks when 95% (75% on testnet) of the network has upgraded:
```

Comment:
Mhmm, I think putting those in ContextualCheckBlock and Consensus::GetFlags (not existing yet, see https://github.com/jtimon/bitcoin/commit/9ba7d0641019d7f6c08f7b15ce3a2bd2e5e8a4ac) would be more informative. But anyway, it seems I squashed the loop change too fast, I can separate that again if it's going to delay this.